src/administration/web/configure-environment.md

```diff
@@ -21,7 +29,49 @@ git push origin :BRANCH-NAME
 > **note**
 > Deleting the Master environment is forbidden.
 
-## Basic Authentication / Restrict access to IPs
+### Outgoing emails
+
+From this tab, you can allow your application to send emails via a SendGrid SMTP proxy. 
+
+![env email](/images/env_email.png)
+
+Changing this setting will temporarily list the environment's status as "Dirty". Once it has re-deployed, it will appear once again as "Active" in your settings.
```

Comment:
Are we still using Dirty in the UI?  I thought that was changed.

If it still says "Dirty", then we should add after it:

```
as "Dirty", meaning its status is changing.
```

Reply:
It has been switched to "Building"